newsfragments/1660.bugfix.rst

```diff
@@ -0,0 +1 @@
+- Added formatter rules for eth_tester middleware to allow getBalance by using integer block numbers
```

Comment:
Oh the `-` isn't necessary here -- towncrier adds that on its own. You only need to add one if you have *sub*-bullets for a particular issue. It might be nice to link to :meth:`~web3.Eth.getBalance` here. Do the docs mention being able to use block number? If not, adding to the docs would be good to :+1:

Reply:
I believe its clear enough that a block identifier [can be used](https://web3py.readthedocs.io/en/stable/web3.eth.html#web3.eth.Eth.getBalance) here. There's a lot of contextual hints about block identifiers in the docs, but I'm realizing that some more formal definition could be helpful, with the options for specifying them. Noted in my notebook; I'll look for an opportunity to add it if one doesn't already exist.